### DISCUSSION TITLE
Lost the name of the radio station from media_title:

### DISCUSSION THREAD

**Discussion by sp0ngeboobs** [music-assistant/support#2210] [Q&A] [answered]
I don't know if this is a bug or a new data format. But after the latest updates (2024.05, b147), the radio station name is missing from the "media_title:" attributes of the player. Now the name of the song is transferred there. And it is duplicated in "stream_title:"

Now I have the name of the song twice in the player :)

![Screenshot_1](https://github.com/music-assistant/hass-music-assistant/assets/147326667/aba2d5c2-5bbb-4291-a0dc-c9efe057b67d)
![Screenshot_2](https://github.com/music-assistant/hass-music-assistant/assets/147326667/c2fbe768-e7cd-4a7d-b5b6-7e804ff9c596)

When I just listened to a song from Soundcloud, "media_title:" gave the full title of the song and "stream_title:" was "null". With soundcloud, everything remained ok, as it was before the updates.

**Comment by OzGav** [comment]
What is the URL to the station?

**Comment by sp0ngeboobs** [comment]
all my stations that I added via URL to the "Radio" section. Previously, in "media_title:" I received the name of the station that I set in MA, now there is the name of the song.
examples of stations. 
https://strm112.1.fm/deephouse_mobile_mp3
https://cast.mediaonline.net.ua/terrace320
https://relay2.radiotoolkit.com/spdeep

**Reply by OzGav** [reply, marked as answer]
Just closing the loop on this. This would be expected behaviour. The media_title should be the name of the track. Some online stations send info that is put in stream_title. The station name is put into the album_name attribute.
